configs/cm3_defconfig

```diff
@@ -23,6 +23,10 @@ BR2_PACKAGE_HOST_MTOOLS=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS=y
 BR2_PACKAGE_HOST_GENIMAGE=y
 
+# Bridging Eth to USB
+BR2_PACKAGE_USB_BRIDGE=y
+BR2_SYSTEM_DHCP="br0"
```

Comment:
below in the the same file you declared eth0 as the BR2_SYSTEM_DHCP. Remove the one for eth0.